google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/Exceptions.java

```diff
@@ -106,12 +141,32 @@ public static StorageException toStorageException(
     if (error == null) {
       return null;
     }
+    String streamName = error.getEntity();
+    String errorMessage =
```

Comment:
Add a comment about this is to trim the entity sub message. Also narrow it further down by looking for "Entity: ".

Reply:
I cannot just look for "Entity:" as of now since it is somehow missing from the detailMessage of `OFFSET_ALREADY_EXISTS` error. The full detailMessage I get looks like: "ALREADY_EXISTS: The offset is within stream, expected offset 1, received 0" (instead of "ALREADY_EXISTS: The offset is within stream, expected offset 1, received 0 Entity: projects/.../datasets/.../tables/testtable/streams/CiQ2NGY3OT...").

Reply:
So why do you need to trim the next message?

Reply:
What does "next message" mean? I need to trim the message of whitespaces either way (with or without Entity) in order to get the "expected offset [0-9]+, received [0-9]+" portion to parse out the offset numbers.